tests/Facebook/InstantArticles/Transformer/GlobalHtml/global.html

```diff
@@ -14,6 +14,9 @@ <h1>The <b>article</b> title</h1>
                 <img src="http://domain.com/image-body.png"/>
                 <span class="caption">The caption for image.</span>
             </div>
+            <div class="hero-image">
```

Comment:
Consider adding a caption block of text to be added to the image on the header.
This way we will cover not only images, but also caption to these images.